automatic/tightvnc/tightvnc.nuspec

```diff
@@ -29,8 +29,4 @@ With TightVNC, you can:
     <tags>remote control vnc admin</tags>
     <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages</packageSourceUrl>
   </metadata>
-  <files>
-    <file src="tools\**" target="tools" />
-  </files>
-</package>
-<!-- “UTF-8” test -->
```

Comment:
that test would be:
```xml
<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
```

just add ^^ right above the package start element (or readd the old UTF-8 test)